components/ui/status-indicator.tsx

```
@@ -0,0 +1,22 @@
+    return (
+        <div
+            className={cn(
+                "size-1 rounded-full",
+                variant === "current" &&
+                    "bg-emerald-500 shadow-emerald-500 animate-pulse-glow-green",
+                variant === "wip" &&
+                    "bg-yellow-500 shadow-yellow-500 animate-pulse-glow-yellow",
+                className
+            )}
+            aria-label={variant === "current" ? "Current" : "Work in Progress"}
+        />
```

Biome (2.4.4)

[error] The ARIA attribute 'aria-label' is not supported by this element.

(lint/a11y/useAriaPropsSupportedByRole)